server/controllers/userController.js

```diff
@@ -0,0 +1,77 @@
+import userModel from '../models/users';
+import encypt from '../helpers/encrypt';
+
+const { encryptPassword, decryptPassword, generateToken } = encypt;
+
+/**
+ *
+ *
+ * @export
+ * @class UsersController
+ */
+export default class UsersController {
+  /**
+   * Signup middleware - Create User Account
+   * @static
+   * @param {object} req
+   * @param {object} res
+   * @param {function} next
+   * @returns {object} Object containing token to the user
+   * @memberof UsersController
+   */
+  static async signUp(req, res, next) {
+    try {
+      const data = req.body;
+      const { password } = data;
+      data.password = await encryptPassword(password);
```

Comment:
why distructure data on `line 25` and still make use of `data.password` on` line 26`

Reply:
This was implemented in the branch I pulled to work on. But password was destructured from data so that it can be encrypted before being sent into the db.